internal/proposals/logs/dynamic-backend-configuration/overview.md

```diff
@@ -0,0 +1,149 @@
+# Motivation
+Kyma uses Fluent Bit as a log collection solution. Fluent Bit is configured with a plain text file (or multiple files) and does not support dynamic configuration.
+It poses the following problems:
+
+1. Static configuration should be provided upon Kyma installation and cannot be changed afterward.
+2. It's impossible to separate the Kyma system configuration from the customer one.
+3. Every time we change the system configuration in managed Kyma, we have to touch Kyma Environment Broker's code.
+
+# Goal
+Come up with a way to dynamically configure Fluent Bit (no manual pod restarting required). It should be possible to apply additional configuration snippets, which are not resettable by the Kyma upgrade process (reconciliation).
+
+# Possible solutions
+What immediately comes to mind is Prometheus Operator, which solves very similar problems. 
+Prometheus, same as Fluent Bit, is configured with a plain text file (possibly, very long and complex). Configuring Prometheus by hand is not a trivial task.
+Prometheus Operator simplifies the configuration by introducing a bunch of custom resources translatable in a Prometheus config.
+It also makes it possible to dynamically update different parts of a Prometheus config without restarting any pods.
+
+After some investigation, the following open source solutions have been found:
+
+- Fluent Bit Operator from Kubesphere: https://github.com/kubesphere/fluentbit-operator
+- Logging Operator from Banzai Cloud: https://github.com/banzaicloud/logging-operator
+
+## Kubesphere Operator
+Fluent Bit Operator defines the following custom resources:
+* `FluentBit`: Defines Fluent Bit instances and its associated config. (it requires kubesphere/fluent-bit for dynamic configuration.)
+* `FluentBitConfig`: Select input/filter/output plugins and generates the final config into a Secret.
+* `Input`: Defines input config sections.
+* `Parser`: Defines parser config sections.
+* `Filter`: Defines filter config sections.
+* `Output`: Defines output config sections.
+Each Input, Parser, Filter, Output represents a Fluent Bit config section, which are selected by FluentBitConfig via label selectors. The operator watches those objects, makes the final config data, and creates a Secret to store it, which will be mounted onto Fluent Bit instances owned by Fluent Bit.
```

Comment:
```suggestion
Each Input, Parser, Filter, Output represents a Fluent Bit config section, which are selected by FluentBitConfig using label selectors. The operator watches those objects, makes the final config data, and creates a Secret to store it, which will be mounted onto Fluent Bit instances owned by Fluent Bit.
```

"using" instead of "via", Kyma [docu guidelines](https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology) insist :)

Reply:
It's good to know! ;)